plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -257,6 +257,7 @@ class WC_Install {
 		),
 		'9.1.0' => array(
 			'wc_update_910_add_launch_your_store_tour_option',
+			'wc_update_910_add_old_refunded_order_items_to_product_lookup_table',
```

Comment:
WC dev newbie question: since 9.1 is releasing soon, should this be updated to `920` instead?

Reply:
Hi @ianlin
Good call! It should say:
```
'9.2.0' => array(
		'wc_update_920_add_launch_your_store_tour_option',
),
```

Since code freeze will happen this week, it will probably be 9.3.0.

I think this fix shouldn't be released alone. I would release it with the other fixes needed to address the problems related to Net sales.

Reply:
> I think this fix shouldn't be released alone

I will create a feature branch then.